client/app/src/Controller/Report/DocumentController.php

```diff
@@ -405,55 +405,6 @@ public function summaryAction(Request $request, int $reportId)
         ];
     }
 
-    /**
```

Comment:
Is this route never mentioned anywhere? I can't see any places where you've changed the routing to bypass this.